com.ibm.wala.cast.python.ml/data/tensorflow.xml

```diff
@@ -786,6 +792,58 @@
       </class>
     </package>
 
+    <package name="tensorflow/distribute">
+      <class name="MirroredStrategy" allocatable="true">
```

Comment:
I want to focus on https://www.tensorflow.org/api_docs/python/tf/distribute/Strategy#run, but I can't tell where it is. These seem to be intertwined. Are they part of a class hierarchy? Is that why?